slasher/rpc/server.go

```diff
@@ -150,3 +150,30 @@ func (ss *Server) IsSlashableBlock(ctx context.Context, req *ethpb.SignedBeaconB
 	return psr, nil
 
 }
+
+// IsSlashableAttestationNoUpdate returns true if the attestation submitted
+// is a slashable vote(no db update is being done).
+func (ss *Server) IsSlashableAttestationNoUpdate(ctx context.Context, req *ethpb.IndexedAttestation) (*slashpb.Slashable, error) {
+	slashings, err := ss.detector.DetectAttesterSlashings(ctx, req)
+	sl := &slashpb.Slashable{Slashable: false}
```

Comment:
Do you think you could move this higher up? Also no need to declare it if its going to be false `Slashable{}` would default to false, so you could just return `Slashable{Slashable: true}` and the end.

Reply:
changed. thanks